src/OpenDebugAD7/AD7DebugSession.cs

```diff
@@ -1302,12 +1304,24 @@ protected override void HandleContinueRequestAsync(IRequestResponder<ContinueArg
 
         protected override void HandleStepInRequestAsync(IRequestResponder<StepInArguments> responder)
         {
-            StepInResponse response = new StepInResponse();
+            try
+            {
+                var granularity = responder.Arguments.Granularity.GetValueOrDefault();
+                StepInternal(responder.Arguments.ThreadId, enum_STEPKIND.STEP_INTO, granularity, AD7Resources.Error_Scenario_Step_In);
+                responder.SetResponse(new StepInResponse());
+            }
+            catch (AD7Exception e)
+            {
```

Comment:
>{ [](http://example.com/codeflow?start=12&length=1)

Shouldn't this have `responder.SetError(new ProtocolException(e.Message));`?

Reply:
Thx done!